dados/DadosAlunos.java

```diff
@@ -17,7 +17,7 @@ public static List<Aluno> getAlunosCadastrados() {
     }
 
     public static void adicionarAluno(Aluno aluno) {
-        if (alunoEhCadastrado(aluno)) {
+        if (!alunoEhCadastrado(aluno)) {
```

Comment:
O método irá jogar uma exceção toda vez que o aluno não for cadastrado, o oposto do que queremos.
- O método deve jogar a exceção apenas quando já houver um aluno cadastrado
- Tirar a exclamação no **if** irá resolver o problema